instrumentation/sidekiq/test/helpers/mock_loader.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'sidekiq/cli'
+require 'sidekiq/launcher'
+
+class MockLoader
```

Comment:
This class is mimicking some of the behaviour found here https://github.com/mperham/sidekiq/blob/master/bin/sidekiqload